Request/RequestBodyParamConverter20.php

```diff
@@ -16,7 +16,7 @@
 
 /**
  * This code is needed for SensioFrameworkExtraBundle 2.x compatibility
- * https://github.com/FriendsOfSymfony/FOSRestBundle/issues/622
+ * https://github.com/FriendsOfSymfony/FOSRestBundle/issues/622.
```

Comment:
I guess this can be removed in 2.0 but we can leave that to another PR .. but then we should revert this change.